client/src/core/client/admin/components/ModerationReason/ModerationReason.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FunctionComponent, useCallback, useState } from "react";
+
+import { Box, Button, Flex } from "coral-ui/components/v2";
+
+import { RejectCommentReasonInput } from "coral-stream/__generated__/RejectCommentMutation.graphql";
+
+import { Localized } from "@fluent/react/compat";
```

Comment:
Can you please clean up the import organization at the top of this file?

Reply:
✅